website/docs/podman/podman-remote.md

```diff
@@ -0,0 +1,112 @@
+---
+sidebar_position: 21
+title: Remote access
+description: Podman Desktop can access remote instances of Podman.
+tags: [podman, installing, windows, macOS]
+keywords: [podman desktop, containers, podman, remote]
+---
+
+# Remote access
+
+Podman Desktop can manage remote Podman connections. This is facilitated through a list of connections using the command `podman system connection ls`.
+
+Containers can be created, started, stopped, and deleted as if managed locally.
+
+This functionality is enabled by connecting via SSH to the Podman socket on the remote host.
+
+**Ed25519** keys, an **SSH** connection, and an enabled **Podman Socket** are required for remote access.
+
+RSA keys are _not supported_; Ed25519 keys are the recommended and only current method to set up a remote connection.
```

Comment:
Do we have the upstream issue we could add as a footnote here ?

Reply:
https://github.com/mscdex/ssh2/issues/1375

Reply:
Thanks @benoitf